vars/tmsUpload.groovy

```diff
@@ -94,7 +94,11 @@ void call(Map parameters = [:]) {
 
         def namedUser = jenkinsUtils.getJobStartedByUserId()
 
-        if (config.useGoStep == true) {
+        def useGoStep = config.useGoStep
+        if (useGoStep == null) {
+            useGoStep = true
+        }
```

Comment:
```suggestion
        def useGoStep = config.useGoStep :? true
```

Reply:
Thanks for the hint! Done with [77b67a5](https://github.com/SAP/jenkins-library/pull/4535/commits/77b67a5eba457529f2224f5654274e548afed913), but the operator should be `?:`, I guess.